aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/AnalyticsPinpointInstrumentedTest.java

```diff
@@ -50,11 +51,8 @@
  * Validates the functionality of the {@link AmazonPinpointAnalyticsPlugin}.
  */
 public class AnalyticsPinpointInstrumentedTest {
+    private static final Logger LOG = Amplify.Logging.forCategory(CategoryType.ANALYTICS);
```

Comment:
This drops the `aws-` prefix from the log slugs, which doesn't necessarily seem wrong to me, but did spark the thought that is there a reason for individual plugins to differentiate themselves? (say, for instance, you're using the `aws-analytics-pinpoint` plugin and the `foo-analytics-bar` plugin, for example.

Reply:
Yea, I'm not sold on it either. I'll update the PR to only include the default log level change.